config/environments/development.rb

```diff
@@ -36,11 +36,19 @@
 
   config.action_mailer.perform_caching = false
 
-  config.action_mailer.delivery_method = :mailgun
+  config.action_mailer.delivery_method = :smtp
   config.action_mailer.mailgun_settings = {
     api_key: ENV["MAILGUN_API_KEY"] ,
     domain: 'sandbox4f9920610a894b81b82f6bc37e90f1a0.mailgun.org',
   }
+  config.action_mailer.smtp_settings = {
+    :authentication => :plain,
+    :address => "smtp.mailgun.org",
+    :port => 587,
+    :domain => "sandbox4f9920610a894b81b82f6bc37e90f1a0.mailgun.org",
+    :user_name => "postmaster@MYDOMAIN.mailgun.org",
```

Comment:
This should be `postmaster@sandbox4f9920610a894b81b82f6bc37e90f1a0.mailgun.org`

Reply:
fixed